changelog.txt

```diff
@@ -1,4 +1,5 @@
 0.77
+ * Realm.allObjects(), RealmResults.sort() and RealmQuery.findAll() extended with multi-field sorting.
```

Comment:
Perhaps "Realm.allObjects(), RealmResults.sort() and RealmQuery.findAll() now supports multi-field sorting" instead?